<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1218</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1218</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
    <addaction name="actionUpload"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPlatforms"/>
   <addaction name="actionComponents"/>
   <addaction name="actionConfigurator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionPlatforms">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Платформы</string>
   </property>
  </action>
  <action name="actionConfigurator">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Конфигуратор</string>
   </property>
  </action>
  <action name="actionComponents">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Комплектующие</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Сохранить данные</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Загрузить данные</string>
   </property>
  </action>
  <action name="actionUpload">
   <property name="text">
    <string>Выгрузить данные для клиентов...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
